moped-database/docker-compose.yml

```diff
@@ -1,9 +1,26 @@
 # Architecture Independent Docker Stack Components for Moped Development Environment
 
 version: '3.7'
+services:
+    hasura:
+        image: hasura/graphql-engine:latest
```

Comment:
Hasura's images of `graphql-engine` are built multi-arch so they run natively on ARM and X86 chips alike. No need for us to manually build an ARM image anymore. 😓